server/src/main/java/org/apache/druid/server/http/CoordinatorResource.java

```diff
@@ -158,4 +164,28 @@ public Response getStatusOfDuties()
   {
     return Response.ok(new CoordinatorDutyStatus(coordinator.getStatusOfDuties())).build();
   }
+
+  @GET
+  @Path("/brokerConfigurationStatus")
```

Comment:
This API should be moved `CoordinatorDynamicConfigResource` class, and path should be

```
/druid/coordinator/v1/config/syncedBrokers
```

or something similar